src/App.js

```diff
@@ -1,10 +1,14 @@
-import NavBar from './components/NavBar';
-import HomePage from './components/HomePage';
-import './App.css';
+import NavBar from "./components/NavBar";
+import HomePage from "./components/HomePage";
+import SignInPage from "./components/authentication/SignInPage";
+// import SignUpPage from "./components/authentication/SignUpPage";
```

Comment:
WHy are these commented ?

Reply:
so i had to comment them out because netlify if there is a warning the netlify action on the PR will not pass. if I don't comment it out it will fail due to it being an unused import.

Reply:
Why aren't we using those ?

Reply:
speaking to u rn